kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/configuration/NMSettingsConverter.java

```diff
@@ -531,6 +553,22 @@ public static Map<String, Variant<?>> buildVlanSettings(NetworkProperties props,
                 .orElse(new ArrayList<String>()), listType));
         return settings;
     }
+    
+    public static Map<String, Variant<?>> buildEthernetSettings(NetworkProperties props, String deviceId) {
+        Map<String, Variant<?>> settings = new HashMap<>();
+        Integer mtu = getIp4MtuFromProperties(props, deviceId);
+        logger.info("REMOVEME-- eth mtu {}", mtu);
+        settings.put("mtu", new Variant<>(new UInt32(mtu)));
+        return settings;        
+    }
+
+    public static Map<String, Variant<?>> buildLoopbackSettings(NetworkProperties props, String deviceId) {
+        Map<String, Variant<?>> settings = new HashMap<>();
+        Integer mtu = getIp4MtuFromProperties(props, deviceId);
+        logger.info("REMOVEME-- lo mtu {}", mtu);
+        settings.put("mtu", new Variant<>(new UInt32(mtu)));
+        return settings;
+    }
```

Comment:
As we said, we don't support Loopback configuration, therefore this can be removed.

```suggestion
```